src/test/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplicationTest.java

```diff
@@ -156,12 +155,13 @@ public void testSetMemory() {
 
         entity.setMemory(CUSTOM_MEMORY);
         assertEquals(entity.getAttribute(VanillaCloudFoundryApplication.ALLOCATED_MEMORY).intValue(), CUSTOM_MEMORY);
-        verify(location, times(1)).setMemory(APPLICATION_NAME, CUSTOM_MEMORY);
+        //TODO: verify(location, times(1)).setMemory(APPLICATION_NAME, CUSTOM_MEMORY);
```

Comment:
Currently, `CloudFoundryPaasLocation#setMemory()` requires the `applicationName` the `memory` and the `localArtifactPath`. When, the `cf-java-client` [issue#544](https://github.com/cloudfoundry/cf-java-client/issues/544) will be addressed, the `CloudFoundryPaasLocation#setMemory()` will be called avoiding the `localArtifactPath`